start.sh

```diff
@@ -219,7 +202,14 @@ setupCouchbase() {
     N1QLAPI=$(getIpPort couchbase 8093)
     echo
     echo 'Creating Couchbase buckets'
-    removeBucket benchmark
+    while true; do
+        echo -n '.'
+        curl -sf -u ${COUCHBASE_USER}:${COUCHBASE_PASS} \
```

Comment:
I was trying to make a minimal intervention in this PR so that we're focusing on telemetry. There's a couple things I'd like to clean up, but I didn't want to go too far down the rabbit hole. Up to you though.